sound/soc/intel/boards/sof_da7219_max98373.c

```diff
@@ -72,6 +72,7 @@ static const struct snd_kcontrol_new controls[] = {
 static const struct snd_kcontrol_new m98360a_controls[] = {
 	SOC_DAPM_PIN_SWITCH("Headphone Jack"),
 	SOC_DAPM_PIN_SWITCH("Headset Mic"),
+	SOC_DAPM_PIN_SWITCH("Spk"),
```

Comment:
is this a mono amp?

Reply:
This is a two mono amp configuration, but there is only a single instance of the amp driver to control the speaker enable pin, similar to the application of max98357a. The shutdown offers more power saving than the auto-suspend.